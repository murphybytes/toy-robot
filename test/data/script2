MOVE
PLACE 0,0,EAST
REPORT
MOVE
REPORT
LEFT
MOVE
REPORT
MOVE
MOVE
MOVE
MOVE
REPORT
LEFT
LEFT
MOVE
REPORT
RIGHT
MOVE
REPORT
MOVE
REPORT
LEFT
LEFT
REPORT
MOVE
MOVE
MOVE
MOVE
REPORT
