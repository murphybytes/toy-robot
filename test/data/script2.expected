EAST (x,y) (0, 0)
EAST (x,y) (1, 0)
NORTH (x,y) (1, 1)
NORTH (x,y) (1, 4)
SOUTH (x,y) (1, 3)
WEST (x,y) (0, 3)
WEST (x,y) (0, 3)
EAST (x,y) (0, 3)
EAST (x,y) (4, 3)
